app/src/main/java/com/techlads/composetv/features/home/navigation/topbar/HomeTopBar.kt

```diff
@@ -0,0 +1,152 @@
+import androidx.tv.material3.TabRowScope
+import androidx.tv.material3.Text
+import com.techlads.composetv.R
+import com.techlads.composetv.features.home.leftmenu.data.MenuData
+import com.techlads.composetv.features.home.leftmenu.model.MenuItem
+import com.techlads.composetv.features.home.navigation.NestedScreens
+import com.techlads.composetv.features.settings.screens.profile.ProfilePicture
+
+@OptIn(ExperimentalTvMaterial3Api::class)
+@Composable
+fun HomeTopBar(
+    content: @Composable () -> Unit,
+    selectedId: String = MenuData.menuItems.first().id,
+    onMenuSelected: ((menuItem: MenuItem) -> Unit)?
+) {
+    var selectedTabIndex by remember { mutableIntStateOf(0) }
+
+    Column {
+        Row(
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically,
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(24.dp)
+        ) {
+            TabRow(selectedTabIndex = selectedTabIndex,
+                indicator = { _, _ ->
+                    Box(
+                        modifier = Modifier
+                            .clip(MaterialTheme.shapes.small)
+                            .background(Color.Transparent)
+                    )
+                }
+            ) {
+                NavigationTabItem(item = MenuData.settingsItem,
+                    isSelected = selectedId == MenuData.settingsItem.id,
+                    onMenuSelected = {
+//                        selectedTabIndex = index
+                        onMenuSelected?.invoke(MenuData.settingsItem)
+                    })
+                MenuData.topBarMenuItems.forEachIndexed { index, menuItem ->
+                    NavigationTabItem(
+                        item = menuItem,
+                        isSelected = selectedId == menuItem.id,
+                        onMenuSelected = {
+                            selectedTabIndex = index
+                            onMenuSelected?.invoke(it)
+                        })
+                }
+            }
+            Text(
+                text = stringResource(id = R.string.app_name),
+                style = MaterialTheme.typography.bodyLarge.copy(fontWeight = FontWeight(700))
+            )
+        }
+        Box(
+            modifier = Modifier
+                .fillMaxSize()
+        ) {
+            content()
+        }
+    }
+}
+
+@OptIn(ExperimentalTvMaterial3Api::class)
+@Composable
+fun TabRowScope.NavigationTabItem(
+    item: MenuItem,
+    isSelected: Boolean,
+    enabled: Boolean = true,
+    onMenuSelected: ((menuItem: MenuItem) -> Unit)?
+) {
+    Tab(
+        selected = isSelected, enabled = enabled,
+        onFocus = {
+            onMenuSelected?.invoke(item)
+        },
+        colors = TabDefaults.pillIndicatorTabColors(
+            selectedContentColor = Color.Black,
+            focusedSelectedContentColor = if (item.id != NestedScreens.Settings.title) Color.Black else Color.White
```

Comment:
We will not hardcode colors, it will affect later when we change the theme, so I will rather request checking other places where we have use material theme colors for example surface etc